ignore.list

```diff
@@ -407,3 +407,23 @@
 // When browser.urlbar.autocomplete.enabled is false, the dropdown is disabled
 // Setting the preference to 0 effectively disables the Location Bar dropdown entirely
 user_pref("browser.urlbar.maxRichResults",			0);
+
+
+// privacy.clearOnShutdown.passwords was removed in Firefox 42
+// https://hg.mozilla.org/releases/mozilla-esr45/rev/fb4f42bbeb01
+"privacy.clearOnShutdown.passwords"
+
+// Old Firefox 3.1a2 pref
+// https://hg.mozilla.org/mozilla-central/rev/adb1ef78dd21
+"browser.sessionstore.enabled"
+
+// PREF: Enable Information Bar for Outdated Plugins (Firefox < 33)
+// http://forums.mozillazine.org/viewtopic.php?f=8&t=2490287
+// https://hg.mozilla.org/mozilla-central/rev/189803
+"plugins.hide_infobar_for_outdated_plugin"
+
+// PREF: Do not store POST data in saved sessions (Firefox < 29)
+// https://hg.mozilla.org/mozilla-central/rev/163997
+// http://kb.mozillazine.org/Browser.sessionstore.postdata
+// relates to CIS 2.5.7
+user_pref("browser.sessionstore.postdata",			0);
```

Comment:
Well I just cut-pasted it from user.js and th CI system doesn't complaint about it. As long as the pref name is between double quotes it should be fine. I can change it to just `"browser.sessionstore.postdata"` if it's important.